src/jdk.jlink/share/classes/jdk/tools/jlink/resources/jlink.properties

```
@@ -127,6 +127,7 @@ err.runtime.link.packaged.mods=This JDK has no packaged modules.\
 err.runtime.link.modified.file={0} has been modified
 err.runtime.link.patched.module=jlink does not support linking from the run-time image\
 \ when running on a patched runtime with --patch-module
+err.all.module.path.empty.mod.path=ALL-MODULE-PATH requires --module-path option (or --module-path does not exist)
 err.empty.module.path=empty module path
```

`err.empty.module.path` is unused.

```suggestion
err.no.module.path=--module-path option must be specified with --add-modules ALL-MODULE-PATH 
err.empty.module.path=No module found from the module path {0}
```
```

---

I've updated it and mentioned `ALL-MODULE-PATH` in the error of a non-existing module path.